guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -10,14 +10,21 @@ functional_areas:
   - Setup
   - Configuration
 ---
+The ece-tools package, also known as _ece-tools_, is compatible with Magento version 2.1.4 and later to provide a rich set of features you can use to manage your {{site.data.var.ee}} project.  It contains scripts and {{site.data.var.ece}} commands designed to help manage your code and automatically build and deploy your Cloud projects.
```

Comment:
I think we need to be more precise in how we're referring to products in this paragraph.

- Use the Commerce variable to refer to Magento 2.1.4
- Use the Magento Commerce (Cloud) variable to refer to managing your projects
- Drop the "Cloud" reference at the end of the sentence

> "The `ece-tools` package, also known as _ece-tools_, is compatible with ~Magento~ {{site.data.var.ee}} version 2.1.4 and later to provide a rich set of features you can use to manage your ~{{site.data.var.ee}}~{{site.data.var.ece}} project.  It contains scripts and commands designed to help manage your code and automatically build and deploy your projects.